rfcs/0084-ZHF-on-master-and-freeze-dates.md

```diff
@@ -0,0 +1,104 @@
+---
+feature: nixos-freeze-dates
+start-date: 2021-01-17
+author: Jonathan Ringer (@jonringer)
+co-authors:
+related-issues: [NixOS release schedule](https://github.com/NixOS/rfcs/pull/80)
+---
+
+# Summary
+[summary]: #summary
+
+To further minimize the amount of risk going into a NixOS release, I propose
+to set freeze dates regarding changes to critical packages being merged to
+staging. In addition, I also propose to stabilize the release on master
+instead of the release branch, thus avoiding the need to backport every pull
+request, and take advantage of the stabilization that unstable normally
+receives.
+
+# Definitions
+[definitions]: #definition
+
+- "Freezing" a branch or package.
+  - Disallow breaking changes to branch or package. In SemVer terminology, disallow major version bumps.
+  - A release branch can be thought of being permanently "frozen" in this regard. Stable may be a better term, but it's already overloaded with a previous release branch or channel.
```

Comment:
I'm not sure if we should define such a detail here: from my observation this isn't the status quo and IMHO also a fairly debatable, but out-of-scope topic.

Reply:
This was more meant to help convey that freezing only applied to breaking changes. No expects the stable channel to receive 0 updates, but people do expect the stable channel to rarely cause upgrade issues.